backup-cronjob.yaml

```
@@ -12,8 +12,8 @@ spec:
           containers:
           - command:
             - /bin/sh
-            - /scripts/backup.sh
-            image: ghcr.io/adfinis-adsygroup/openshift-etcd-backup
+            - /usr/local/bin/backup.sh
+            image: ghcr.io/adfinis/openshift-etcd-backup
```

image path needs to match our orga name:
```suggestion
            image: ghcr.io/adfinis-adsygroup/openshift-etcd-backup
```